blackcat/blackcat/storysharing/templates/storysharing/display_story.html

```diff
@@ -46,6 +46,9 @@
             {% elif editable and storywriter.active and story.available and not form %}
                 <p class="bottom links">Thank you for adding your snippet! Wait for one of the other writers to add theirs...</p>
             {% endif %}
+            {% if story.shareable or editable %}
```

Comment:
This message should only be displayed to the writers of the story, not to the public, even if the story us shareable.